lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -4,4 +4,77 @@
             data-user-create-subcomment="<%= canCreateSubComment %>">
         <a href="#" class="new-post-btn" role="button"><span class="icon fa fa-edit new-post-icon"></span><%= gettext("New Post") %></a>
     </div>
+
+    <div class="team-detail">
+        <h4><%- gettext("Team Details") %></h4>
+        <% if (isMember) { %>
+            <div class="team-member">
+                <span><%- gettext("You are a member of this team.") %></span>
+            </div>
+        <% } %>
+        <div class="team-members">
+            <span class="sr"><%- gettext("Team members profile") %></span>
```

Comment:
Should this be "Team member profiles" instead? (i.e. "profiles" is the plural word)